### ISSUE TITLE
 The PayPal web payment is not going through.

### ISSUE THREAD

**Issue by woo94** [paypal/paypal-ios#290]
### PayPal SDK Version

1.4

### Environment

Sandbox

### Xcode Version

Xcode 15.4

### OS Version & Device

iPhone 13, iOS 17.5.1

### Integration type

CocoaPods

### Development Processor

Apple Silicon (M-series chips)

### Describe the bug

I tried to use PayPal for payment, but since the account hasn't been used for a while, it was logged out for account protection. Please return to the seller and restart the payment process.

And when I tried pressing the PayPal button again, I got an error like the one in the picture.

I thought about creating a different account to try, but I can't log out, so I can't move on to the next step.

### To reproduce

Just press the PayPal button.

### Expected behavior

.

### Screenshots

![IMG_8560](https://github.com/user-attachments/assets/4bae673b-0600-48f6-b390-1d8d6a83dad8)
![IMG_8561](https://github.com/user-attachments/assets/dfb19f27-05b8-413d-941d-6cf65b258fec)

**Comment by KunJeongPark** [comment]
Thank you for using our PPCP SDK.
This appears to be issue with PayPal log in page which the SDK simply presents in a webView.
Please contact Developer Support for this issue (https://www.paypal-support.com/s/?language=en_US)

**Comment by woo94** [comment]
@KunJeongPark Thanks for the reply. I appreciate👍

I also contacted developer support regarding this issue. Since my code flow is no different from the demo app, I thought the cause of the problem might be the order. So, I created a new order and proceeded with the payment process, and I was able to log in without any issues. It would be great if appropriate error handling could be implemented so that the developer can identify and address the cause.

And also i have one question.

I would like to know how long a created order can be used.
